lib/SILGen/LValue.h

```diff
@@ -524,7 +524,6 @@ struct LLVM_LIBRARY_VISIBILITY ExclusiveBorrowFormalAccess : FormalAccess {
   ExclusiveBorrowFormalAccess &
   operator=(ExclusiveBorrowFormalAccess &&) = default;
 
-  ExclusiveBorrowFormalAccess() = default;
```

Comment:
Why this change?

Reply:
The superclass doesn't have a default constructor, so the subclass can't either.